pkgs/by-name/ni/nixos-rebuild-ng/src/nixos_rebuild/process.py

```diff
@@ -127,10 +128,12 @@ def run_wrapper(
         kwargs,
         extra_env,
     )
+    # This makes error messages more readable by not having Path objects in the error message.
+    stringified_args = [str(arg) if isinstance(arg, Path) else arg for arg in args]
```

Comment:
This requires to update our mock tests.

Reply:
I think my suggestion of just handling the exception in `main()` will avoid the need to change the mocks too.